app/src/app/api/v0/emissions-factor/route.ts

```diff
@@ -1,15 +1,56 @@
 import { db } from "@/models";
 import { apiHandler } from "@/util/api";
 import createHttpError from "http-errors";
-import { NextResponse } from "next/server";
-import { Op } from "sequelize";
+import { NextRequest, NextResponse } from "next/server";
+import { Op, WhereOptions } from "sequelize";
+import { EmissionsFactor } from "@/models/EmissionsFactor";
 
-export const GET = apiHandler(async (_req: Request, _context: {}) => {
+export const GET = apiHandler(async (req: NextRequest, _context: {}) => {
   // don't return emissions factors from specific inventories
+  const whereClause: WhereOptions<EmissionsFactor> = {
+    inventoryId: { [Op.is]: null },
+  };
+
+  // TODO uncomment after ON-2210 is merged
```

Comment:
Uncomment the code, mark PR as draft, add dependency and link to the relevant PR to this PR's description.

Thanks :pray: :sparkles: